python/instrumentation/openinference-instrumentation-bedrock/examples/bedrock_example.py

```diff
@@ -49,3 +48,55 @@
         response = client.invoke_model(modelId="anthropic.claude-v2", body=prompt)
     response_body = json.loads(response.get("body").read())
     print(response_body["completion"])
+
+def converse_example():
+    message1 = {
+            "role": "user",
+            "content": [{"text": "Create a list of 3 pop songs."}]
+    }
+    message2 = {
+            "role": "user",
+            "content": [{"text": "Make sure the songs are by artists from the United Kingdom."}]
```

Comment:
nit: coud maybe consider having the first message be a "system" (don't know if claude has system messages) message or something (common in prompting) and message 2 be a user message and simplify the example a bit while still showing the multiple messages

system_mes
```suggestion
   system_message = { "role": "system", "content": [{"text": "You are an expert at creating music playlists"}]}
   user_message = {"role": "user", "content": [{"text": "can you create a list of 3 pop songs by artists from the United Kingdom"}]}
```